travis-scripts/jbossws-test.sh

```diff
@@ -33,12 +33,13 @@ if [[ $USE_WFLY_MASTER = "true" ]]; then
   cd wildfly
 
   # compile in silence.  The bld output is too much for travis log
-  mvn --quiet clean install -DskipTests -Denforcer.skip -Dcheckstyle.skip
+  mvn --quiet clean install -DskipTests -Denforcer.skip -Dcheckstyle.skip -Prelease
 
   WFLY_TARGET=`pwd`"/dist/target/"
   WFLY_HOME=$(find $WFLY_TARGET -maxdepth 1 -name \wildfly\* -type d -print | head -n1)
   D_SERVER_HOME="-Dserver.home="$WFLY_HOME
   cd $MYPWD
+  mvn -s .travis-settings.xml -B -fae -DSECMGR=${SECURITY_MGR} -P${SERVER_VERSION} ${D_SERVER_HOME} integration-test
```

Comment:
Why these lines was added instead of the original one? The practical result is the same, just more lines.